src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlagTracing.cs

```diff
@@ -29,18 +28,28 @@ internal class FeatureFilterTracing
         public bool UsesPercentageFilter { get; set; } = false;
         public bool UsesTimeWindowFilter { get; set; } = false;
         public bool UsesTargetingFilter { get; set; } = false;
-       
+        public bool UsesSeed { get; set; } = false;
+        public bool UsesTelemetry { get; set; } = false;
+        public bool UsesVariantConfigurationReference { get; set; } = false;
+        public int MaxVariants { get; set; }
+
         public bool UsesAnyFeatureFilter()
         {
             return UsesCustomFilter || UsesPercentageFilter || UsesTimeWindowFilter || UsesTargetingFilter;
         }
 
-        public void ResetFeatureFilterTracing()
+        public bool AnyTracingFeaturesUsed()
+        {
+            return UsesSeed || UsesTelemetry || UsesVariantConfigurationReference;
+        }
+
+        public void ResetFeatureFlagTracing()
```

Comment:
You need to reset maxvariants right? And what about uses telemetry and uses variant configuration reference?